packages/client/src/components/LeaderboardTable.re

```diff
@@ -53,7 +53,7 @@ module Columns = {
       ~render=
         (r: record) =>
           switch (r.pointScore) {
-          | Some(pointScore) => pointScore |> string_of_float |> Utils.ste
+          | Some(pointScore) => pointScore |> Js.Float.toString |> Utils.ste
```

Comment:
why? I prefer the E. library, so this could be standardized.

Reply:
string_of_float is deprecated

Reply:
Ah, can you put the functionality in the E library then, so all of our library use is one place, especially for ``Float``? Like, ``E.Float`` or whatever it is.

Reply:
yes, thanks

Reply:
Fixed.